README.md

```diff
@@ -1 +1,66 @@
-# module-cosmosdb
\ No newline at end of file
+# Ballerina Azure Cosmos DB Endpoint
+
+The Cosmos DB endpoint allows you to access the Cosmos DB through Ballerina.
```

Comment:
```suggestion
The Cosmos DB endpoint allows you to access the Cosmos DB service through Ballerina.
```

Reply:
@MaryamZi thoughts? It's wrong to say Cosmos DB in singular form here. Is the above suggestion ok?

Reply:
Shouldn't it be 
```suggestion
The Cosmos DB endpoint allows you to access Cosmos DB through Ballerina.
```